it/google-cloud-platform/src/test/java/org/apache/beam/it/gcp/cloudsql/CloudSqlResourceManagerTest.java

```diff
@@ -88,6 +88,11 @@ private MockCloudSqlResourceManager createTestManager(boolean useCustomDb) {
           public @NonNull CloudSqlResourceManager build() {
             return new MockCloudSqlResourceManager(this);
           }
+
+          @Override
+          protected void configurePort() {
+            this.setPort(1234);
```

Comment:
Why do we need this? Is it just to silence the warning or are we depending on a static port number?

Reply:
Since I am testing an abstract class, I have to implement the abstract methods, so I just created a dummy method. But it also serves the purpose of ensuring the correct port is passed in the tests which isn't necessarily trivial given that it is based on the Testcontainers framework which generates random port by default (not what we want for this static-based RM)